src/NuGet.Clients/PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -339,7 +339,7 @@ public sealed class UIActionEngine
             // don't show this dialog for VS 2015
             return await Task.FromResult(true);
 #else
-            var newProjectNames = new List<string>();
+            var projectsCouldChange = new List<NuGetProject>();
```

Comment:
nit: it took me a while to figure out what this name was referring to. how about ``nonNuGetProjects`` or ``potentialProjects``?